nebula_messages/continental_srvs/srv/ContinentalSrr520SetRadarParameters.srv

```diff
@@ -0,0 +1,15 @@
+bool autoconfigure_extrinsics 1
+float32 longitudinal
+float32 lateral
+float32 vertical
+float32 yaw
+float32 pitch
+float32 vehicle_wheelbase -1.0
+
+uint8 sensor_id # The new sensor id
+float32 cover_damping # Whether the plug is in the bottom
+bool plug_bottom # Cover damping in dB
```

Comment:
Comments are switched :D

Reply:
Addressed in [8f61ccc](https://github.com/tier4/nebula/pull/124/commits/8f61ccc3efafccb8808c3b5cfa2d710b45a9fc8f)
:bow: